src/compile/mark/encode/zindex.ts

```diff
@@ -9,7 +9,13 @@ export function zindex(model: UnitModel) {
   const order = encoding.order;
 
   if (!isPathMark(mark) && isValueDef(order)) {
-    return wrapCondition(model, order, 'zindex', cd => signalOrValueRef(cd.value));
+    return wrapCondition({
+      model,
+      channelDef: order,
+      vgChannel: 'zindex',
+      mainRefFn: cd => signalOrValueRef(cd.value),
+      invalidValueRef: undefined // zindex encoding doesn't have continuous scales and thus can't have invalid values
```

Comment:
FYI, no behavior change here. I just switch `wrapCondition` to use parameter argument and make it explicit that `zindex` won't need the new `invalidValueRef` added.